libs/ml/src/ops/matrix_multiply.cpp

```diff
@@ -285,6 +285,25 @@ std::vector<typename fetch::math::SizeType> MatrixMultiply<T>::ComputeOutputShap
   return output_shape;
 }
 
+template <typename T>
+MLChargeAmount MatrixMultiply<T>::ChargeForward()
+{
+  assert(!this->batch_input_shapes_.empty());
+
+  // TODO(VH): impl. for n-dimensional case, not only for 2D.
```

Comment:
if you are not going to do this implementation in this PR you need to raise a task for it and put the task number in the TODO

Reply:
Done, ML-482